### ISSUE TITLE
Gentoo Repository Broken

### ISSUE THREAD

**Issue by acidtonic** [ros/ros-overlay#865]
Tried just about everything with regards to syncing and playing with use flags.
```

emerge: there are no ebuilds to satisfy "net-libs/ignition-transport:3=".
(dependency required by "sci-electronics/gazebo-7.11.0::ros-overlay" [ebuild])
(dependency required by "ros-kinetic/gazebo_dev-2.5.19-r1::ros-overlay" [ebuild])
(dependency required by "ros-kinetic/gazebo_plugins-2.5.19-r1::ros-overlay" [ebuild])
(dependency required by "ros-kinetic/gazebo_ros_pkgs-2.5.19-r1::ros-overlay" [ebuild])
(dependency required by "ros-kinetic/simulators-1.3.2::ros-overlay" [ebuild])
(dependency required by "ros-kinetic/desktop_full-1.3.2::ros-overlay" [ebuild])
(dependency required by "ros-kinetic/desktop_full" [argument])

```

**Comment by flabrosse** [comment]
Is that why some packages are missing, such as ros-lunar/laser_filters or ros-lunar/rospack?

**Comment by allenh1** [comment]
> Is that why some packages are missing, such as ros-lunar/laser_filters or ros-lunar/rospack?

@flabrosse those are missing because `ros-lunar` is now EOL.

It is preserved by a tag, however. You can checkout `lunar_eol` to grab those ebuilds.

**Comment by allenh1** [comment]
> Tried just about everything with regards to syncing and playing with use flags.

@acidtonic I'll try and figure out what's going on with that.

**Comment by flabrosse** [comment]
> > Is that why some packages are missing, such as ros-lunar/laser_filters or ros-lunar/rospack?
> 
> @flabrosse those are missing because `ros-lunar` is now EOL.
> 
> It is preserved by a tag, however. You can checkout `lunar_eol` to grab those ebuilds.

That is interesting.  I had not realised.  In fact, when I checked laser_filters, to check it was still supported in lunar, lunar does not appear as one of the eol's.  I guess time to move on...

**Comment by allenh1** [comment]
> In fact, when I checked laser_filters, to check it was still supported in lunar, lunar does not appear as one of the eol's

Yeah, it's not always entirely clear. [This listing](http://wiki.ros.org/Distributions) is usually quite reliable, for future reference.

**Comment by acidtonic** [comment]
I'm here and working on this now if you need me to test any changes.
